app/assets/stylesheets/style.scss

```diff
@@ -132,112 +131,19 @@ body {
     width: 1000px;
     text-align: center;
     top: 50%;
-    animation: move_up .5s 1.9s forwards;
+    img {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3